app/src/bcsc-theme/features/account/components/NicknameForm.tsx

```diff
@@ -93,6 +99,7 @@ const NicknameForm: React.FC<NicknameFormProps> = ({ isRenaming }) => {
         text2: t('Unified.NicknameAccount.RenameSuccessToastMessage'),
         position: 'bottom',
       })
+      await handleUpdateRegistration(trimmedAccountNickname)
```

Comment:
Why not just
```suggestion
      await registration.updateRegistration({ ...store.bcsc, selectedNickname: trimmedAccountNickname })
```

Then you don't need the extra function declaration added above

Reply:
Also should we catch here as well? It's possible for this call to throw

Reply:
Yes it is possible to throw, I will wrap with try/catch and align with the comment above